pandas/tests/io/parser/test_common.py

```diff
@@ -2179,3 +2179,13 @@ def test_read_csv_names_not_accepting_sets(all_parsers):
     parser = all_parsers
     with pytest.raises(ValueError, match="Names should be an ordered collection."):
         parser.read_csv(StringIO(data), names=set("QAZ"))
+
+
+def test_read_csv_with_use_inf_as_na(all_parsers):
+    # https://github.com/pandas-dev/pandas/issues/35493
```

Comment:
see if u can locate this test near others that set options (not kwargs)

Reply:
The others that use options seem to be in more targeted testing files (test_to_html.py, test_printing.py); this is the only one in this file

Reply:
kk